modules/hal_nordic/nrfs/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_NRFS)
+    add_subdirectory(${ZEPHYR_CURRENT_MODULE_DIR}/nrfs nrfs)
+    if(CONFIG_NRFS_LOCAL_DOMAIN)
```

Comment:
`nrfs_backend_ipc_service_lite.c` sounds like `CONFIG_NRFS_BACKEND_IPC_SERVICE` and not `CONFIG_NRFS_LOCAL_DOMAIN`. Not quite sure where is a problem, but it is `nrfs_backend_ipc_service_lite.c` that uses the `NRFS_BACKEND_IPC_SERVICE` dependencies.

Reply:
fixed